src/components/Tables/Table.tsx

```diff
@@ -3,9 +3,15 @@
 import React, { useState, useEffect } from "react";
 import DataTable, { TableColumn } from "react-data-table-component";
 import TableFilterBar, { FilterText } from "@/components/Tables/TableFilterBar";
+import { styled } from "styled-components";
+import { NoSsr } from "@mui/material";
```

Comment:
nice, didn't know this was a thing, was planning on finding a `NoSsr` thing at some point!